qiskit_ibm_runtime/runtime_job.py

```diff
@@ -485,9 +485,13 @@ def _error_msg_from_job_response(self, response: Dict) -> str:
         job_result_raw = self._download_external_result(
             self._api_client.job_results(job_id=self.job_id())
         )
-        index = job_result_raw.rfind("Traceback")
-        if index != -1:
-            job_result_raw = job_result_raw[index:]
+
+        try:
+            index = job_result_raw.rfind("Traceback")
+            if index != -1:
+                job_result_raw = job_result_raw[index:]
+        except TypeError:
+            pass
```

Comment:
@cpkurotori What do you think about this? Should we add some sort of try/except 

Or check the type with something like:

```python
if isinstance(job_result_raw, bytes):
     job_result_raw = job_result_raw.decode('utf-8')
```

Reply:
honestly i think it would be better to change `_download_external_result` (https://github.com/Qiskit/qiskit-ibm-runtime/pull/1220/files#diff-f14680ef8724b0c65249d7b84df515bd3db9ce534b509e373580eacc6400609bR169)

```diff
- return result_response.content
+ return result_response.text
```

Reply:
Wasn't aware of the difference, thanks for the suggestion